index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/style-main.css">
+    <title>Setup and Mobile Version Skeleton</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400;500&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+
+    <header>
+        <nav class="navbar">
+            <img class="toggleNav" src="/images/ic_menu.svg" alt="hamburguer menu icon">
+            <ul>
+                <li><a href="#">Hello</a></li>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+
+        <section class="headline">
+
+            <div class="headline-top">
+
+                <h1 class="headline-name">
+                    Eduardo <br>
+                    Sancho
+                </h1>
+
+                <p class="headline-bio">
+                    Hello! I am a software developer! <br>
+                    I can help you build a product, feature or <br>
+                    website. Take a look at my work. <br>
+                    If you like what you see and have a project you <br>
+                    need coded, don’t hesitate and contact me.
+                </p>
+
+                <button id="start-collaboration" type="button">Start collaboration</button>
+
+            </div>
+
+            <div class="headline-last">
+                <img id="shapes" src="/images/image_geometry_2.png" alt="beautiful colorful shapes">
+            </div>
```

Comment:
- [ ] Please consider deleting this `img` tag and applying the image as a background image in your `CSS` as was requested by the previous reviewer.

> As per the design, kindly delete this image and use it as a background instead.
Hint: You can create an empty div, and apply this image as background, to ease positioning